Sync WPT with upstream (17-02-2020)

servo-wpt-sync opened the pull request:
Automated downstream sync of changes from upstream as of 17-02-2020.
[no-wpt-sync]
r? @servo-wpt-sync

Commit:
Update web-platform-tests to revision ac16628eb7eb601957382053011363d2bcf8ce44

servo-wpt-sync commented:
@bors-servo r+

bors-servo commented:
:pushpin: Commit 7e7c887 has been approved by `servo-wpt-sync`

<!-- @bors-servo r=servo-wpt-sync 7e7c8873e443f5960d23785d250e472027a0eef3 -->

bors-servo commented:
:hourglass: Testing commit 7e7c8873e443f5960d23785d250e472027a0eef3 with merge 4f2b9424606db708106c2637ee5afdd4397b82ae...

highfive commented:
Heads up! This PR modifies the following files:
 * @jgraham: tests/wpt/webgl/meta/MANIFEST.json

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/JUDsumBRQYW3tUX4XGrfhA)

CYBAI commented:
https://github.com/servo/servo/pull/25764#issuecomment-586277731 https://github.com/servo/servo/pull/25764#issuecomment-586278492

CYBAI committed:
Update to use assert_throws_* functions for mozilla tests

CYBAI committed:
Diff manifests with raw string

After WPT moves to "Manifest path trie RFC", we don't have the `paths`
field in manifest anymore.

But, we still have file hash in the `items` field; the simplest way to
diff between manifests is to compare the raw string value of the `items`
field. When we have different manifests, we must have different items
field in the manifest json.

CYBAI commented:
The fix for both issues is at servo-wpt-sync/servo#1

jdm commented:
@bors-servo r+

bors-servo commented:
:pushpin: Commit 55f0d5a has been approved by `jdm`

<!-- @bors-servo r=jdm 55f0d5ade1ce18b98e0d7817e16a8673ee6967c5 -->

jdm commented:
```

Running the WPT lint on ./tests/wpt/mozilla/tests/...
{"path": "mozilla/promise.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 28, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_worklet.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 21, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_worklet.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 25, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_worklet.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 29, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_worklet.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 33, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 21, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 25, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 29, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 33, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 37, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 41, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 45, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 49, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 53, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 57, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/worklets/test_paint_worklet_loading.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 61, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/service-workers/service-worker-registration.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 45, "rule": "PROMISE_REJECTS"}
{"path": "mozilla/service-workers/service-worker-registration.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 49, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/filter-does-not-match.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 102, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/name-empty-device-from-name-prefix-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/name-missing-device-from-name-wrong-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/adapter-not-present.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/name-empty-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/blocklisted-service-in-optionalServices.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/blocklisted-service-in-optionalServices.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/name-empty-device-from-name-wrong-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/single-filter-two-services-fails.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/not-accept-all-devices-without-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/name-missing-device-from-name-prefix-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/no-devices.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/blocklisted-service-in-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/adapter-off.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/name-missing-device-from-name-empty-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/device-not-found-with-extra-data.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 59, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/device-not-found-with-service-and-manufacturer-data.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 10, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/empty-namePrefix.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 34, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/max-length-for-device-name-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 10, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/max-length-for-device-name-namePrefix.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 10, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/wrong-mask-length.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 66, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/empty-filters-member.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/unicode-max-length-for-device-name-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 12, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/blocklisted-service-data-key.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 49, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/no-arguments.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/empty-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/filters-xor-acceptAllDevices.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/unicode-max-length-for-device-name-namePrefix.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 12, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/wrong-manufacturer-data-key.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 70, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/wrong-service-in-optionalServices-member.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 34, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/empty-services-member.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 34, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/wrong-service-in-services-member.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 33, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/wrong-service-data-key.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 84, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/canonicalizeFilter/no-filters-member.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/acceptAllDevices/optional-services-missing.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 11, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/requestDevice/acceptAllDevices/accept-all-devices-with-filter.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 9, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/connect/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/notify-failure.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/characteristic-does-not-support-notifications.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/service-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/blocklisted-characteristic.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/startNotifications/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/disconnect-called-during-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/descriptors-not-found-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 28, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 30, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/descriptors-not-found.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/disconnect-called-before-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/blocklisted-descriptors-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/invalid-descriptor-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/characteristic-is-removed-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/device-goes-out-of-range-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptors/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/stopNotifications/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/stopNotifications/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/stopNotifications/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 20, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/descriptor/blocklisted-descriptor.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 20, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/descriptor/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/descriptor/service-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/descriptor/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/descriptor/descriptor-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/descriptor/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 20, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/characteristic/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/characteristic/service-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/characteristic/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/characteristic/value-too-long.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/characteristic/blocklisted-characteristic.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/writeValue/characteristic/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/descriptor/blocklisted-descriptor.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/descriptor/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/descriptor/service-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/descriptor/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/descriptor/descriptor-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/descriptor/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 20, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/characteristic/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/characteristic/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/characteristic/service-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/characteristic/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/characteristic/blocklisted-characteristic.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/readValue/characteristic/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/reconnect-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/invalid-characteristic-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/service-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/blocklisted-characteristic.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 26, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 27, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 28, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 29, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 30, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 31, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 32, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/characteristic-not-found.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristic/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnect-called-during-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnected-device.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 12, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/blocklisted-services-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/invalid-service-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 13, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/services-not-found-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 27, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 28, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 29, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/services-not-found.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnected-device-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 12, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnect-called-before-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/no-permission-present-service.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 13, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/no-permission-absent-service-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/no-permission-absent-service-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/no-permission-absent-service-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/no-permission-present-service-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/no-permission-present-service-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/no-permission-present-service-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/device-goes-out-of-range-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryServices/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/blocklisted-descriptor.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/characteristic-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/descriptor-not-found.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 18, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/invalid-descriptor-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 27, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 28, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getDescriptor/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 19, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/disconnected-device.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 12, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/invalid-service-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 13, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/service-not-found.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/no-permission-present-service.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/no-permission-present-service.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/no-permission-present-service.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 25, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 26, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/disconnect-invalidates-object.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 27, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/no-permission-absent-service.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/no-permission-absent-service.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/no-permission-absent-service.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getPrimaryService/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-called-during-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/service-is-removed-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/invalid-characteristic-name.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 15, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-called-during.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/blocklisted-characteristics-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 28, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 29, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 30, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 31, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 32, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 33, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-invalidates-objects.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 34, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/blocklisted-characteristics.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-called-before-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/service-is-removed.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/device-goes-out-of-range.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/characteristics-not-found-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/characteristics-not-found.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 14, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/device-goes-out-of-range-with-uuid.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 16, "rule": "PROMISE_REJECTS"}
{"path": "bluetooth/getCharacteristics/disconnect-called-before.html", "message": "Test-file line has a `promise_rejects(...)` call", "lineno": 17, "rule": "PROMISE_REJECTS"}
```

@bors-servo r-

jdm commented:
I'm fixing these locally.

jdm committed:
Remove uses of promise_rejects.

jdm commented:
@bors-servo r+

bors-servo commented:
:pushpin: Commit 3a9e2a6 has been approved by `jdm`

<!-- @bors-servo r=jdm 3a9e2a677d6f330fb121720d037581152ecd5f11 -->

bors-servo commented:
:hourglass: Testing commit 3a9e2a677d6f330fb121720d037581152ecd5f11 with merge f0c2c2764b10f189ffe0046078fa0536ac566406...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/VV1LccheRya3PVjQlXgKEQ)

jdm commented:
@bors-servo try=wpt

bors-servo commented:
:hourglass: Trying commit d1a1f007ed556193104a10162cda30af69b2f7f3 with merge 2f7a873741d4136d3ba30c497e927efad0fb16af...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/SjcDvemWSqK_wgY8fqljKQ)

jdm committed:
Remove intermittent failure.

jdm commented:
@bors-servo try=wpt

bors-servo commented:
:hourglass: Trying commit dc051715684e1758b6d7b726acbad66968bc4714 with merge 6cc713aa08b9bd30c962db3134f19383a133d894...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/T3cmaYpyRDCeVkcZxLejRQ)

jdm commented:
@bors-servo r+

bors-servo commented:
:pushpin: Commit dc05171 has been approved by `jdm`

<!-- @bors-servo r=jdm dc051715684e1758b6d7b726acbad66968bc4714 -->

bors-servo commented:
:hourglass: Testing commit dc051715684e1758b6d7b726acbad66968bc4714 with merge cda975459967010b7f75b007edf9ac091c435793...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/eS-qCW-RQ1y08OFzTi11hg)

jdm commented:
@bors-servo retry
* #24726 #25808 #25805

bors-servo commented:
:hourglass: Testing commit dc051715684e1758b6d7b726acbad66968bc4714 with merge 5d523918734d7666cc1b832379d0354108ea6212...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/Rh-HsFq_RsaspSosKQO_nQ)

CYBAI commented:
@bors-servo retry

- #24762, #25513 

```
  ▶ PASS [expected FAIL] /css/css-text/line-breaking/line-breaking-atomic-003.html
```

bors-servo commented:
:hourglass: Testing commit dc051715684e1758b6d7b726acbad66968bc4714 with merge e562816f465d91ae654e30e983116a0eaa98c053...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/cuIPKYUTR7iuWWg_ng_u6w)

CYBAI commented:
Looks like they were intermittent results 👀

```
  ▶ PASS [expected FAIL] /css/css-text/line-breaking/line-breaking-atomic-003.html
  ▶ FAIL [expected PASS] /css/css-text/white-space/trailing-other-space-separators-break-spaces-015.html
  │   → /css/css-text/white-space/trailing-other-space-separators-break-spaces-015.html 633b086c40bb5edfd5368340d4944d2581614865
  └   → /css/css-text/white-space/reference/trailing-other-space-separators-break-spaces-002-ref.html 99268a97df84a57aee9aa1d974cdbc902c3156de
```

jdm commented:
@bors-servo retry

bors-servo commented:
:hourglass: Testing commit dc051715684e1758b6d7b726acbad66968bc4714 with merge 14f78dbb7eb0c97826b646af5300c55dffbc16e2...

bors-servo commented:
:broken_heart: Test failed - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/YOzexENTRAm8utPBLWDT_g)

jdm commented:
@bors-servo retry
* #25812 #25811

bors-servo commented:
:hourglass: Testing commit dc051715684e1758b6d7b726acbad66968bc4714 with merge f5ff38b87559cd25db44fa7bc6f2a9c4cb2264e0...

bors-servo commented:
:sunny: Test successful - [status-taskcluster](https://community-tc.services.mozilla.com/tasks/groups/NBCJgG6HQMalJzIy-zI5RA)
Approved by: jdm
Pushing f5ff38b87559cd25db44fa7bc6f2a9c4cb2264e0 to master...
